src/dictionaries/words

```diff
@@ -0,0 +1,204 @@
+app
+apps
+args
+auth
+authenticator
+bar
+baz
+behavior
+behaviour
+boolean
+booleans
+colored
+coloured
```

Comment:
Like the other terms under discussion, I think we should decide one way or the other (US or not) and be consistent. It  looks a bit shonky having a random selection of multiple "correct" spelling.

Reply:
It's horrid, but I propose putting it in like this and creating a fix issue, because I agree, it's not nice.

Reply:
That's fine with me (I guess we have to delete the new dictionary entries *and* fix every instance in the docs?)